commands/command.go

```diff
@@ -27,19 +28,25 @@ func (cmd *BaseCommand) Before(c *cli.Context) error {
 	util.LoggerInit(c.GlobalBool("verbose"))
 	cmd.out = util.Logger()
 	cmd.machine = Machine{Name: c.GlobalString("name"), out: util.Logger()}
+
+	util.NotifyInit(fmt.Sprintf("Outrigger (rig) %s", c.App.Version))
+	cmd.context = c
+
 	return nil
 }
 
 // Success encapsulates the functionality for reporting command success
 func (cmd *BaseCommand) Success(message string) error {
 	if message != "" {
 		cmd.out.Info.Println(message)
+		util.NotifySuccess(cmd.context, message)
```

Comment:
Since a number of our commands do not return cmd.Success(), is the idea that not returning Success() means the command is opting out of notifications? It seems as though we should return success for everything and have a boolean parameter to suppress notifications.

Reply:
They should all be returning cmd.Success now, that was part of the PR to transition to this centralized system.  If you dont want a message or notifications you just return `""` the empty string.

Reply:
Okay. We may want to switch this per my comment later on if we have other things triggered on Success that we don't want commands to opt-out from. That seems fine to happen when such a use case arises.